File: IKMS-server/src/main/java/pl/politechnika/ikms/rest/dto/register/RegisterEntryDto.java
package pl.politechnika.ikms.rest.dto.register;

import lombok.Data;
import pl.politechnika.ikms.rest.dto.MinimalDto;

import java.time.LocalDateTime;

@Data
public class RegisterEntryDto {

    private MinimalDto<Long, String> child;

    private String description;

    private LocalDateTime created;

    private MinimalDto<Long, String>  employee;

    private Long registerId;
}

File: IKMS-client/ngapp/src/app/employeeSiteModule/model/diary/registerEntryDto.ts
export class RegisterEntryDto {
    child: any;
    description: string;
    created: any;
    employee: any;
    registerId: number;
    
    constructor() {
        this.child = null;
        this.description = '';
        this.created = null;
        this.employee = null;
        this.registerId = null;
    }
}